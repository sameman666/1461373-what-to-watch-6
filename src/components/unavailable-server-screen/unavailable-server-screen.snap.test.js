import React from 'react';
import {render} from '@testing-library/react';
import UnavailableServerScreen from './unavailable-server-screen';

it(`Should Unavailable Server Screen render correctly`, () => {
  const {container} = render(<UnavailableServerScreen />);
  expect(container).toMatchSnapshot();
});
