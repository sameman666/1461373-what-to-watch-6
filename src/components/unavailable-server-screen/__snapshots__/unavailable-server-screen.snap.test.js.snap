// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Unavailable Server Screen render correctly 1`] = `
<div>
  <div
    class="page-content"
    style="min-height: 100vh; text-align: center;"
  >
    <h1>
      Сервер временно не доступен
    </h1>
  </div>
</div>
`;
